onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -330,6 +330,7 @@ class PlannerImpl {
     for (auto it = freelist_.begin(); it != freelist_.end(); ++it) {
       size_t reusable = static_cast<size_t>(it->ml_value);
       const onnxruntime::NodeArg* p_node_arg = ort_value_info_.at(reusable).p_def_site;
+      if (!p_node_arg) continue;
```

Comment:
is continuing the correct behavior here? or should this be an error?

Reply:
This looks like an error case to me... p_node_arg should always be populated... @souptc to confirm

What's reason for updating this? Are you seeing any test failure because of this ?

Reply:
yes, something like this: https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=137690
when running onnx_test_runner with "-o 0 -e dml -j 1 <path to downloaded test data>\models\opset8\test_vgg19", p_node_arg is null

Reply:
It is an error case ... it should not be null at this point. So, the real bug must be earlier.